src/questions/fjord-foundry.yaml

```diff
@@ -0,0 +1,170 @@
+- uuid: fed2ef38-e906-4e1b-93bf-d4ec5bdf3ac3
```

Comment:
Add a questions

- Which of these is not a configuration option while creating a new LBP?